portfoliostyle.css

```diff
@@ -0,0 +1,391 @@
+	margin: 0;
+	text-align: left;
+	padding: 100px;
+}
+
+h2, h3 {
+	text-align: center;
+	background-color: white;
+}
+
+h2 {
+	margin-top: 10vh;
+	font-size: 40px;
+	border-radius: 30px;
+	padding: 20px 0;
+}
+
+h3 {
+	color: black;
+	border-radius: 15px;
+	padding: 5px 0;
+}
+
+p a {
+	color: white;;
+}
+
+footer div {
+	border-top: 2px solid gray;
+	padding: 10px 0;
+	text-align: right;
+}
+
+/* most class styles */
+
+.nav-item {
+	color: white;
+	padding: 10px;
+	text-align: center;
+}
+
+.project-item {
+	display: flex;
+	justify-content: space-evenly;
+	padding: 20px;
+	background-color: gray;
+	color: white;
+	margin: 20px 0;
+	border-radius: 15px;
+}
+
+.project-item a {
+	padding: 10px;
+	background-color: lightgray;
+	border-radius: 5px;
+	color: black;
+	border-left: 2px solid slateblue;
+	border-top: 2px solid slateblue;
+	text-decoration: none;
+}
+
+.project-item a:hover {
+	padding: 10px;
+	background-color: black;
+	border-radius: 5px;
+	color: lightblue;
+	text-decoration: none;
+}
+
+.project-image {
+	max-width: 200px;
+	height: 150px;
+	margin: auto 0;
+	border: 2px solid lightgray;
+	border-radius: 15px;
+}
+
+.project-info {
+	display: flex;
+	flex-direction: column;
+	width: 45%;
+}
+
+.project-info div {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 10px 0;
+}
+
+.about {
+	padding: 20px 7vw;
+	background-color: gray;
+	color: white;
+	margin: 20px 0;
+	border-radius: 15px;
+}
+
+.nav-item.cv {
+	color: lavender;
+	font-weight: bold;
+}
+
+.nav-item.cv:hover {
+	color: plum;
+}
+
+/* social media buttons*/
+
+.fa {
+	font-size: 50px;
+	width: 100px;
+	height: 100px;
+	text-align: center;
+	text-decoration: none;
+	background-color: black;
+	color: white;
+	border-radius: 40%;
+}
+
+.media-links {
+	display: flex;
+	justify-content: space-evenly;
+	margin-bottom: 30px;
+}
+
+.media-links a {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 10px;
+	padding: 30px;
+}
+
+.media-links a:hover {
+	color: lightblue;
+	border: 3px solid white;
+}
+
+/* timeline css*/
+
+.timeline {
+	position: relative;
+	max-width: 1200px;
+	margin: 0 auto;
+}
+
+.timeline::after {
+  content: '';
+  position: absolute;
+  width: 6px;
+  background-color: white;
+  top: 0;
+  bottom: 0;
+  left: 50%;
+  margin-left: -3px;
+}
+
+.container {
+  padding: 10px 40px;
+  position: relative;
+  width: 50%;
+}
+
+/* the balls on the timeline*/
+.container::after {
+  content: '';
+  position: absolute;
+  width: 25px;
+  height: 25px;
+  right: -17px;
+  background-color: white;
+  border: 4px solid slateblue;
+  top: 15px;
+  border-radius: 50%;
+  z-index: 1;
+}
+
+.left {
+  left: 0;
+}
+
+.right {
+  left: 50%;
+}
+
+.left::before {
+  content: " ";
+  height: 0;
+  position: absolute;
+  top: 22px;
+  width: 0;
+  z-index: 1;
+  right: 15px;
+  border: medium solid white;
+  border-width: 8px 0 8px 25px;
+  border-color: transparent transparent transparent grey;
+}
+
+.right::before {
+  content: " ";
```

Comment:
if you're trying to add in a space write the line:
white-space:pre; 
after the content